### ISSUE TITLE
continue.exe  is a virus!!!?!?!?!? WTF?

### ISSUE THREAD

**Issue by am06** [stephenfewer/grinder#56]
http://www.exeproblems.com/exe-file-error/continue.exe-fix-29881/

Seriously???????? I'd really like to hear an explanation for this!!!

**Comment by stephenfewer** [comment]
continue.exe (SHA1: 3588345B1C76580422C8772C6A16660BAAD31C01) is not a virus. It triggers 2/55 on virustotal [0] probably due to some generic heuristic matching. If you are still concerned you can build it from source and bin diff the two files :)


[0] https://www.virustotal.com/en/file/112cc9eac322c21b8769232d901663dca7814403e86ac634194fe90256bf850d/analysis/

**Comment by am06** [comment]
ok what the hell is this then?

**Comment by am06** [comment]
it made me very paranoid.

**Comment by stephenfewer** [comment]
Fair enough, The source code is in the tree so you can build your own copy to avoid any doubt.

**Comment by am06** [comment]
cheers mate! All good now, I became angry when I googled it and a few guys were talking about the continue.exe virus online! Thanks!

**Comment by stephenfewer** [comment]
No problem, always good to double check these things! :)

**Comment by am06** [comment]
Amen!;)....oh,yeah, one more thing, can we do something to make this work on win XP?

**Comment by stephenfewer** [comment]
It shouldn't be too much work but I do not have the time unfortunately to bring support to older platforms.

**Comment by am06** [comment]
ok, well can you give me some ideas? I got 0 experience with WIN API.

**Comment by stephenfewer** [comment]
Well ./node/source/continue/main.c is pretty simple so any win32 compiler toolchain should be able to compile it. I would either download an old version of MSVC Express or grab something like MinGW and build continue.exe, you should not have to edit the source code.

There is no guarantee the Ruby side will perform as expected on XP so you may hit issues there. Grinder was never tested at any stage on XP, only Win7 and up.

**Comment by am06** [comment]
i've managed to compile that with mingw. I could send it to you if you're interested.
Also you say in the docs that ruby 1.9 is supported but you can't fuzz 64 bit targets with it.......why?

Thanks.
